cmake/BLTMacros.cmake

```diff
@@ -796,7 +796,8 @@ macro(blt_add_executable)
     list(GET arg_SOURCES 0 _first)
     get_source_file_property(_lang ${_first} LANGUAGE)
     if(_lang STREQUAL Fortran)
-        if (NOT CUDA_LINK_WITH_NVCC)
+        #Check to see if HIP is enabled before resetting 
+        if (NOT (CUDA_LINK_WITH_NVCC OR ENABLE_HIP))
```

Comment:
I think since this is after `blt_add_hip_executable` we can be a little more precise by checking `HIP_SOURCE_PROPERTY_FORMAT` on the source file.

Reply:
Ok, let me fix this really quick. Thanks!